Data/models/Domain.swift

```diff
@@ -158,6 +158,26 @@ public final class Domain: NSManagedObject, CRUD {
     }
     return false
   }
+  
+  public static func clearAllEthereumPermissions(_ completionOnMain: (() -> Void)? = nil) {
+    DataController.perform { context in
+      let fetchRequest = NSFetchRequest<Domain>()
+      fetchRequest.entity = Domain.entity(context)
+      do {
+        let results = try context.fetch(fetchRequest)
+        results.forEach {
+          $0.wallet_permittedAccounts = nil
+        }
+      } catch {
+        let fetchError = error as NSError
+        print(fetchError)
+      }
+
+      DispatchQueue.main.async {
+        completionOnMain?()
+      }
+    }
+  }
```

Comment:
can we add a unit test for this static function?

Reply:
[405005e](https://github.com/brave/brave-ios/pull/5258/commits/405005e617494ea5fbe9515bdb740c5b2a5fd6c7)